common/snapshots.py

```diff
@@ -2165,7 +2164,7 @@ def fileInfo(self):
                         continue
                     info = line[:index].strip().split(' ')
                     if len(info) == 3:
-                        d[f] = (int(info[0]), info[1], info[2]) #perms, user, group
+                        d[f.encode('utf-8','replace')] = (int(info[0]), info[1].encode('utf-8','replace'), info[2].encode('utf-8','replace')) #perms, user, group
```

Comment:
Binary mode in Python is something I'm most certainly not too familiar with, but your explanation makes it easy to see how this works. Thanks!